### PR TITLE
add apple silicon

### PR DESCRIPTION
**Pull request by bogyo210**

### CODE REVIEW

File: docs/ci-cd-environment/machine-types.md [hunk 1]
```diff
@@ -79,11 +79,15 @@ Implementation details:
 
 ## Apple machine type
 
-Apple machine types can be paired with the [MacOS Xcode14 image][macos-xcode14] or [MacOS Xcode15 image][macos-xcode15].
+Apple machine type `a1-standard-4`, amd64 based,can be paired with the [MacOS Xcode14 image][macos-xcode14] or [MacOS Xcode15 image][macos-xcode15].
```
Comment (radwo) [inline docs/ci-cd-environment/machine-types.md:82]:
```suggestion
Apple machine type `a1-standard-4`, amd64 based, can be paired with the [MacOS Xcode14 image][macos-xcode14] or [MacOS Xcode15 image][macos-xcode15].
```

File: docs/ci-cd-environment/machine-types.md [hunk 2]
```diff
@@ -79,11 +79,15 @@ Implementation details:
...
+
+Apple machine type `a2-standard-4`, apple silicon based, can be paired with the [MacOS Xcode15 image][macos-xcode15-apple-silicon].
+
 
 | Machine name    | Virtual CPUs <sup>1</sup> | Memory <sup>2</sup> | Disk <sup>3</sup> |
 | --------------- | :-----------------------: | :-----------------: | :---------------: |
 | a1-standard-4   |             4             |          8          |        50         |
+| a2-standard-4   |             4             |          8          |        50         |
```
Comment (radwo) [inline docs/ci-cd-environment/machine-types.md:90]: Previously you mentioned 150 GB disk.

File: docs/ci-cd-environment/machine-types.md (hunk 1 above)
Comment (bogyo210) [reply docs/ci-cd-environment/machine-types.md]: Fixed

File: docs/ci-cd-environment/machine-types.md (hunk 2 above)
Comment (bogyo210) [reply docs/ci-cd-environment/machine-types.md]: Fixed

**Review by radwo — APPROVED** [review]

**Pull request merged**